File: testdata/interprocedural/caller.py
from flask import request
import subprocess

def handle_request():
    cmd = request.args.get("cmd")
    run_command(cmd)

def run_command(user_cmd):
    subprocess.run(user_cmd, shell=True)

def safe_handler():
    data = "static"
    process(data)

def process(value):
    print(value)

File: testdata/interprocedural/caller.js
const { exec } = require('child_process');

function handleRequest(req, res) {
    const cmd = req.query.cmd;
    runCommand(cmd);
}

function runCommand(userCmd) {
    child_process.exec(userCmd);
}

function safeHandler(req, res) {
    const data = "static";
    processData(data);
}

function processData(value) {
    console.log(value);
}
